packages/@react-spectrum/s2/chromatic/DateRangePicker.stories.tsx

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {CalendarDate, CalendarDateTime} from '@internationalized/date';
+import {DateRangePicker} from '../';
+import type {Meta, StoryObj} from '@storybook/react';
+import {userEvent} from '@storybook/test';
+
+const meta: Meta<typeof DateRangePicker> = {
+  component: DateRangePicker,
+  parameters: {
+    chromaticProvider: {disableAnimations: true}
+  },
+  title: 'S2 Chromatic/DateRangePicker'
+};
+
+export default meta;
+
+type Story = StoryObj<typeof DateRangePicker>;
+
+const startDate = new CalendarDate(2022, 2, 3);
+const endDate = new CalendarDate(2022, 2, 10);
+
+export const Default: Story = {
```

Comment:
I get strange styles in the second and third calendars with `visibleMonths={2}` and `visibleMonths={3}`

<img width="902" height="384" alt="Image" src="https://github.com/user-attachments/assets/d80312d3-5a43-4e6d-a60f-17cf2a3404c7" />